ignite-generator/component/templates/component.js.template

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+import React, { ScrollView, View, Text } from 'react-native'
+var styles = require('../Styles/<%= name %>Style')
```

Comment:
we now store styles in the same folder, separate component from container styles.